src/features/surveys/components/SurveyEditor/blocks/TextBlock.tsx

```diff
@@ -96,7 +96,7 @@ const TextBlock: FC<TextBlockProps> = ({
             {header ? (
               element.text_block.header
             ) : (
-              <Msg id="misc.surveys.blocks.text.empty" />
+              <Msg id="misc.surveys.blocks.common.empty" />
```

Comment:
Same as above - i think the empty-messages should be specific to each block, so this should stay as "...blocks.text.empty"